functions/database.inc

```diff
@@ -0,0 +1,51 @@
+<?php
```

Comment:
Do we want this in '/functions'?

I was thinking of making a new direction for backends, that could be '/backends'  The idea is that we would move the neon.inc there as well. then the goal would be for everything in 'functions' to call into those backend functions to access the backends. making '/functions' the middle layer.  

/pages  calls into /functions which calls into /backends?

I have a working case that uses '/functions/datasources' for that layer so that is another options. I just feel like it does not belong in '/functions'

Reply:
I was thinking about that...  I was viewing PDO as the backend, even though it's built into php now.  I figured database.inc, aside of the connection setup, would be commands line the function run() which just makes calling a query easier and we can tweak to point to various PDO filters or other backends (I still like the idea of compiling a PDO backend for NeonCRM, but that's gonna be....  work. :) )  However, I can see how you're suggesting that we move the connection setup to backends, and use database.inc for abstraction.  I'll see if I can easily separate it in the next few minutes.